File: app/src/main/java/com/ludoscity/findmybikes/citybik_es/model/BikeStationExtra.java
package com.ludoscity.findmybikes.citybik_es.model;

public class BikeStationExtra {
    private Boolean locked;
    private String name;

    public BikeStationExtra(boolean locked, String name) {
        this.locked = locked;
        this.name = name;
    }

    public Boolean getLocked() {
        return locked;
    }

    public void setLocked(Boolean locked) {
        this.locked = locked;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/ludoscity/findmybikes/data/database/station/BikeStationExtra.kt
package com.ludoscity.findmybikes.data.database.station

import com.google.gson.annotations.SerializedName

data class BikeStationExtra(val locked: Boolean?, val name: String?, val uid: String?, val renting: Int?,
                            val returning: Int?, val status: String?, @SerializedName("last_updated") val lastUpdated: Long)
